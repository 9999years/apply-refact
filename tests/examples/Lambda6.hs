fun x y z = f g z