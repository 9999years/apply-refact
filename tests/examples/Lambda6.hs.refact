[("tests/examples/Lambda6.hs:1:1-17: Warning: Eta reduce\nFound:\n  fun x y z = f g z\nPerhaps:\n  fun x y = f g\n",[])]
