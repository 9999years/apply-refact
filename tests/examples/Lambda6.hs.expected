fun x y = f g
